planning/technology/storage/README.md

```diff
@@ -66,8 +66,47 @@ This section details the requirements and options specifically for the manuscrip
 
 ##### Potential Partners
 
-1. S3 - storage for git
-2. ElasticSearch - duplicate data to make searchable
+Jack is thinking that really we probably want all of these:
```

Comment:
I agree that using all three of these together is probably going to be our ideal solution, but we'll need to prototype them before we fully commit to make sure that we're not putting ourselves in a world of pain if there is a major limitation.